### PR TITLE
TA 97932

### PR DESCRIPTION
**Pull request by kathancox**
This PR includes the TA and a draft update for the SCRAM docs.

Fixed release not yet published to link on release.

## Preview

https://deploy-preview-16515--cockroachdb-docs.netlify.app/docs/advisories/a97932

### CODE REVIEW

File: advisories/a97932.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Technical Advisory 97932
+advisory: A-97932
+summary: The SCRAM protocol had a high default hash count causing connection latency spikes for clients running on limited CPU. 
+toc: true
+affected_versions: v22.2.0 to v22.2.6
+advisory_date: 2023-03-16
```
Comment (kathancox) [inline advisories/a97932.md:7]: Date here, needs to change when we confirm the publication

**Comment by kathancox** [comment]
@MichaelTrestman This is the TA for the SCRAM hashing issue. I have made a change to the SCRAM page, but this is very much a placeholder/draft. Not sure how much we want to mention this on that page (or location on the page), or if we want to go into more detail re the two cluster settings around this issue. That is, the rehashing and the default scram value settings. 

I can make any changes that you think would fit the situation here. Thanks!

**Review by rimadeodhar — APPROVED** [review]
This LGTM. As mentioned in the comment, lets wait till 22.2.7 is released to publish it.

**Comment by kathancox** [comment]
@MichaelTrestman It looks like Rich is working on this PR https://github.com/cockroachdb/docs/pull/16411/files and might therefore mean those changes are sufficient to cover doc updates for this TA. Rich mentioned he would be adding the new SCRAM value in that PR... Let me know your thoughts as you review.

File: v22.2/security-reference/scram-authentication.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ SCRAM authentication imposes additional computational load on your application s
 
 {% include_cached {{page.version.version}}/scram-authentication-recommendations.md %}
 
+In clusters running v22.2.0 to v22.2.6, the default hashing cost was high for certain clients that are running on a limited amount of CPU, which may cause higher than normal connection latency. See the [Technical Advisory](../../advisories/a97932.html) for mitigation steps.
```
Comment (MichaelTrestman) [inline v22.2/security-reference/scram-authentication.md:56]:
```suggestion
In clusters running v22.2.0 to v22.2.6, the default hashing cost was too high for certain clients running on a limited amount of CPU, causing connection latency. See the [Technical Advisory](../../advisories/a97932.html) for mitigation steps. 
```

**Review by MichaelTrestman — APPROVED** [review]
LGTM. I didn't validate attempting to set the cluster setting to make sure it works (not even that it mitigates the performance issue, but just even that it's a valid cluster setting path and everything)... did you, or are we assuming this works as described?

**Pull request merged**